components/persistor/lib/remote-doc/remote-doc-types/RemoteDocClient.ts

```diff
@@ -1,5 +1,5 @@
 export interface RemoteDocClient {
-    uploadDocument(base64: string, key: string, bucket: string);
+    uploadDocument(base64: string, key: string, bucket: string, contentType: string);
```

Comment:
What are the possible values for contentType? Like what are some examples. Not sure what goes here.

Reply:
added some more details around the types here